app/assets/stylesheets/hackweek.scss

```diff
@@ -42,6 +42,12 @@ img[alt="add-emoji"] {
   }
 }
 
+#similar_projects{
+  border: 1px solid #e9e9e9;
+  border-radius: 1rem;
+  background-image: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSI1IiBoZWlnaHQ9IjUiPgo8cmVjdCB3aWR0aD0iNSIgaGVpZ2h0PSI1IiBmaWxsPSIjZmZmIj48L3JlY3Q+CjxyZWN0IHdpZHRoPSIxIiBoZWlnaHQ9IjEiIGZpbGw9IiNjY2MiPjwvcmVjdD4KPC9zdmc+");
```

Comment:
Why are you using base64 for a background image?

Reply:
just copy pasted from  body tag to make it more appealing to user

Reply:
nop :smile_cat: